.github/dependabot.yaml

```diff
@@ -15,6 +15,14 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+    groups:
+      gomod-breaking:
+        update-types:
+          - major
+      gomod-backward-compatible:
+        update-types:
+          - minor
+          - patch
```

Comment:
Looks like this should go under the `package-ecosystem: "gomod"` section?

Reply:
Oops. Good catch. Should be fixed in 384cf15de492225407e114ed5044f843484d456a